ios/Classes/BetterPlayer.h

```diff
@@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property(nonatomic) float playerRate;
 @property(nonatomic) int overriddenDuration;
 @property(nonatomic) AVPlayerTimeControlStatus lastAvPlayerTimeControlStatus;
+@property(nonatomic) bool isRestorePIPButtonPressed;
```

Comment:
`isRestore` sounds like `re-start` PIP window for me. 
Maybe just `close` or `dismiss` would be suitable in this case.

Reply:
@HiroshiOshiro 
sorry, as I mentioned before 
> This parameter is set to true when the user presses the restore button (on the right side of the PIP).
I think `close` or `dismiss` is more suitable for X button ( on the left side ). Maybe I should change to `isRestoreButtonPIPPressed` or do you have any other recommendations?

Reply:
I see, sorry I misunderstood that.

In that case, it is going back  to our app. If I, I would call the right side button as `backToAppButton` so I think `isBackToAppButtonPressed` would be suitable.

Reply:
@HiroshiOshiro sorry for late, I changed the name, pls check it 🙇